index.html

```diff
@@ -8,6 +8,26 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
+    <center>
+        <header id="header">
+            <img class ="image_geometry_1" src="images/image_geometry_1.png"/>
+           
+            <nav><img class ="ic_menu" src="images/ic_menu.png"/></nav>
+
+        </header>
+
+        <section id="headline">
+            <h1 class="profileName">Harshita Jain</h1>
+            
+            <p class="intro">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me. </p>
+            <button class="startbtn">Start collaboration</button>
+        </section>
+        <img class="geometry" src="images/image_geometry_2.png">
+
```

Comment:
- [ ] [optional] I'm sorry that the previous reviewer did not mention this, kindly place this image as a background (you can use CSS to achieve this). 😄 You can [read more on it](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image).